src/main/java/net/mcreator/element/types/Overlay.java

```diff
@@ -58,10 +58,6 @@ public Overlay(ModElement element) {
 		this.overlayTarget = "Ingame";
 	}
 
-	public boolean hasTextures() {

```

Comment:
More unrelated changes, also seems to cause tests to fail. Please recheck PR in github "Files changed" tab

Reply:
You should also sync changes in WYSIWYG dialogs, they are the same thing.

Reply:
Weird... I have it locally and everything has been pushed to my branch 
![image](https://user-images.githubusercontent.com/38427877/208976736-0946598b-a414-4027-ab5a-eac9a84cb0b3.png)

Reply:
I checked everything and this weird behaviour is, in theory, from the past. Everything I saw was things I changed myself for this PR.